lib/web_of_science.rb

```diff
@@ -22,5 +22,8 @@ def self.queries
     @@queries ||= WebOfScience::Queries.new(client)
   end
 
+  def self.logger
+    @@logger ||= Logger.new(Settings.WOS.LOG)
+  end
```

Comment:
This module doesn't need a logger because each of the classes that it wraps has their own logging.  This logger can be removed.

Reply:
Oh, I see the grand plan is move this responsibility from `NotificationManager` here.  Given the considerable consolidation of responsibility for these things into the `NotificationManager`, moving that responsibility to disparate locations in the app seems to add complexity rather than simplify the app.  In effect, it is not changing the way that `NotificationManager` is called or used to manage the generic `NotificationManager.error` calls; it just stripping the `NotficationManager` of control over how the logger is initialized.  Given all the delegation changes, it sorta makes sense to have the delegations contained in the module namespace.  So, OK, whatever.